src/sentry/static/sentry/app/components/gravatar.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import $ from 'jquery';
 import MD5 from 'crypto-js/md5';
+import Letar from '../components/letar';
```

Comment:
As much as it makes me laugh every time I pronounce "leh-tar", I think we should consider renaming `Letar` to something less ambiguous – for our open source contributors, etc. Something like the far more boring `LetterAvatar`.